test/automation/src/sql/notebook.ts

```diff
@@ -372,31 +372,32 @@ export class NotebookView {
 	 * @returns tree item ids from Notebooks View
 	 */
 	async getNotebookTreeItemIds(): Promise<string[]> {
-		return (await this.code.waitForElements(NotebookView.notebookTreeItem, false)).map(item => item.attributes['id']);
+		return (await this.code.waitForElements(NotebookTreeView.notebookTreeItem, false)).map(item => item.attributes['id']);
 	}
 
 	/**
 	 * Pin the first notebook in the Notebooks View
 	 */
 	async pinNotebook(): Promise<void> {
 		const notebookIds = await this.getNotebookTreeItemIds();
-		await this.code.waitAndDoubleClick(`${NotebookView.notebookTreeItem}[id="${notebookIds[0]}"]`);
-		await this.code.waitAndClick(`${NotebookView.notebookTreeItem}${NotebookView.selectedItem} .codicon-pinned`);
+		// Pinning SQL notebook to prevent the Configure Python Wizard from showing, since Python is no longer set up when the NotebookTreeView test suite starts
+		await this.code.waitAndDoubleClick(`${NotebookTreeView.notebookTreeItem}[id="${notebookIds[1]}"]`);
+		await this.code.waitAndClick(`${NotebookTreeView.notebookTreeItem}${NotebookTreeView.selectedItem} .codicon-pinned`);
 	}
 
 	/**
 	 * Unpin the only pinned notebook.
 	 * Previously pinned by the pinNotebook method.
 	 */
 	async unpinNotebook(): Promise<void> {
-		await this.code.waitAndClick(NotebookView.pinnedNotebooksSelector);
-		await this.code.waitAndClick(`${NotebookView.pinnedNotebooksSelector} .actions a[title="Unpin Notebook"]`);
+		await this.code.waitAndClick(NotebookTreeView.pinnedNotebooksSelector);
```

Comment:
Is this affected by your changes above now since there will be 2 pinned Notebooks? 

(in general I would say this function should be taking in the ID/name of the Notebook to unpin though - having it just pick whatever random one happens to be first could easily result in wrong behavior)